src/hooks/useCategoryFilter.ts

```diff
@@ -273,7 +273,11 @@ function sortCategoryValueViews(
   cvv0: SelectCategoryValueView,
   cvv1: SelectCategoryValueView
 ): number {
-  return COLLATOR_CASE_INSENSITIVE.compare(cvv0.label, cvv1.label);
+  return !cvv0.label
+    ? 1
+    : !cvv1.label
+    ? -1
+    : COLLATOR_CASE_INSENSITIVE.compare(cvv0.label, cvv1.label);
```

Comment:
Filter items with empty labels started showing up at the top rather than the bottom when the underlying value was changed to empty string rather than `null`, so I restored the old behavior by explicitly sorting empty string to the bottom -- not sure if that's important